src/node_worker.cc

```diff
@@ -46,6 +47,8 @@ namespace node {
 namespace worker {
 
 constexpr double kMB = 1024 * 1024;
+std::unordered_map<uint64_t, Worker*> workers;
+MessagePort* main_port;
```

Comment:
Can you please add a comment explaining how access to these values is thread safe? It's not clear to me.

Reply:
I'm awaiting for guidance from @anonrig or other to make sure they actually are, I'm not really familiar with C++ multithreading.

Reply:
You likely need to create a `Mutex` and use `Mutex::ScopedLock` to protect access at all places.
Take care if this happens at places where some other mutex is already acquired to avoid deadlocks.

Reply:
Done, thanks!